app/controllers/posts_controller.rb

```diff
@@ -16,8 +16,8 @@ def show
   end
 
   def new
-    @post = Post.new 
-    @community = Community.find_by(params[:community_id])  
+    @post = Post.new
+    @community = Community.find_by(params[:community_id])
```

Comment:
community find seems to be a common line in many methods, also in other controllers. Can we move it to a separate place from where it can be invoked in all the controllers?

Reply:
here this is necessary as each one is used for different functionalities